report/src/main/kotlin/com/hartwig/actin/report/pdf/util/Tables.kt

```diff
@@ -25,13 +29,33 @@ object Tables {
         return createSingleCol().setWidth(width)
     }
 
+    fun extractTextFromCell(cell: Cell): String {
+        return cell.children
+            .filterIsInstance<Paragraph>()
+            .joinToString("\n") { paragraph ->
+                paragraph.children
+                    .filterIsInstance<Text>()
+                    .joinToString("") { it.text }
+            }
+    }
+
     fun makeWrapping(contentTable: Table, forceKeepTogether: Boolean, skipWrappingFooter: Boolean = false): Table {
-        if (contentTable.numberOfRows == 0) {
-            contentTable.addCell(Cells.createSpanningNoneEntry(contentTable))
+        val tableOnlyContainsFootNote = if (contentTable.numberOfRows == 1) {
+            val text = extractTextFromCell(contentTable.getCell(0, 0))
+            TrialFormatFunctions.FOOTNOTES.any { text.contains(it) }
+        } else false
```

Comment:
It is not very nice that we have to read back a cell in actual code (outside of test code). 

It is a better solution to only add the footnotes if there is content in the first place (so fix the problem when we add the footnote)